### ISSUE TITLE
loss function

### ISSUE THREAD

**Issue by saymamba** [WenRuiCai/HIPTrack#4]
Excellent work! 
I understand from the paper that during the training process, HIPTrack freezes the backbone of OSTrack and only trains the memory module, and uses the loss function of OSTrack. Have you tried to modify the loss function to be more suitable for training the memory module?
Thank you.

**Comment by WenRuiCai** [comment]
Primarily, to ensure the fairness of the comparison and minimize modifications to the original framework, we didn't attempt to use other loss functions in HIPTrack. Therefore, we also employed the end-to-end loss like OSTrack for supervision. However, it's evident that this may not necessarily be the optimal setting. Perhaps loss functions like CIoU or SIoU could achieve better performance. Alternatively, adding additional losses to the memory, such as leveraging contrastive learning loss to enhance different memory features, is also a viable option. We welcome further research and exploration on your part!
